RadixWallet/Core/FeaturePrelude/FactorSourceCard/FactorSourceCard+DataSource.swift

```diff
@@ -87,4 +92,21 @@ extension FactorSourceKind {
 			fatalError("Not supported yet")
 		}
 	}
+
+	var infoLinkContent: (item: InfoLinkSheet.GlossaryItem, title: String) {
+		switch self {
+		case .device:
+			(.device, "Learn about biometrics/PIN")
+		case .ledgerHqHardwareWallet:
+			(.ledger, "Learn about Ledger Nanos")
+		case .offDeviceMnemonic:
+			(.offdevicemnemonic, "Learn about passphrases")
+		case .arculusCard:
+			(.arculuscard, "Learn about Arculus Cards")
+		case .password:
+			(.password, "Learn about passwords")
+		default:
+			fatalError("Not supported yet")
```

Comment:
didn't add the localization for these because we want to use the same key than the glossary item, and I am not sure if they are gonna be named as I did.